app/models/container.rb

```diff
@@ -18,6 +18,7 @@ class Container < ActiveRecord::Base
   has_many   :vim_performance_states, :as => :resource
 
   # Needed for metrics
+  delegate   :ems_id, :to => :container_group
```

Comment:
This feels weird...we try to keep direct relationships to the EMS anyway, so I'm surprised that container doesn't have this already.  For example, every Vm can get to its EMS directly.